src/components/cylc/table/Table.vue

```diff
@@ -145,10 +145,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                     </div>
                     <div class="mr-1">
                       <Job
+                        v-if="item.jobs.length === 0"
                         v-cylc-object="item.node"
                         :status="item.node.state"
                         :previous-state="item.jobs.length > 1 ? item.jobs[1].state : ''"
```

Comment:
Oh, just realized previous-state here should be `''` (empty) actually. No point in checking its length again.

I left these two jobs because I **think** when we don't have jobs we display the state of the task? I think :grimacing:

Reply:
>I think the previous-state is valid; it's the status that should be the latest job instead of the task? I'll push a commit with a simplification

Could be. I was referring to the `v-if="item.jobs.length === 0"` above. If it is added, then it wouldn't make sense to have the `:previous-state="item.jobs.length > 1`, since `item.jobs.length is zero at that point.